### DISCUSSION TITLE
Use Volume Number as Tray Icon

### DISCUSSION THREAD

**Discussion by Riajyuu** [File-New-Project/EarTrumpet#672] [Ideas]
See the title

**Comment by DragoCubed** [comment]
Maybe the volume number could be on the bottom right of the volume icon? It would be hard to tell what it is.

I guess it could just replace the icon if it's an opt in option.

**Comment by treylav** [comment]
I don’t understand why the idea was not supported. I use this type of display for the monitor’s brightness (using ClickMonitorDDC) and it’s very convenient.

![image](https://user-images.githubusercontent.com/617704/82560695-9bc64880-9b7a-11ea-999b-17b31de7528d.png)

**Comment by Riajyuu** [comment]
Nearly 2 years here already. I wonder if there is any plan for this.
